macaw-ppc-symbolic/tools/Dump.hs

```diff
@@ -0,0 +1,23 @@
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE ImportQualifiedPost #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Main (main) where
+
+import Data.ElfEdit qualified as EE
+import Data.Macaw.Dump qualified as MD
+import Data.Macaw.Memory.ElfLoader.PLTStubs qualified as MMEP
+import Data.Macaw.PPC qualified as PPC
+import Data.Macaw.PPC.Symbolic ()
+import Data.Macaw.Symbolic qualified as MS
+import Data.Proxy (Proxy(..))
+
+main :: IO ()
+main = do
+  archVals <-
+    case MS.archVals (Proxy @PPC.PPC32) Nothing of
+      Just archVals -> pure archVals
+      Nothing -> error "impossible"
+  let pltStubInfo :: MMEP.PLTStubInfo EE.PPC32_RelocationType
+      pltStubInfo = error "PLT stub discovery is not supported on PPC"
+  MD.main PPC.ppc32_linux_info archVals pltStubInfo
```

Comment:
Is there a reason to hard-code this to 32-bit PPC here?

Reply:
Just that it would be hard with the current architecture to select between them at the CLI... See most recent commit/message. I made separate executables 32- and 64-bit for now.

Reply:
> What would you think about deferring PPC64 support to an issue?

I'm fine with that.